docs/source/_static/dpf_entry.html

```diff
@@ -0,0 +1,3407 @@
+<p class="p-main" >DPF's CPython interface is based on gRPC service. Its server is available in Workbench installation under dpf/bin/{platform}/Ans.Dpf.Grpc.bat (Windows) or Ans.Dpf.Grpc.sh (Linux). The client is based on a Python SDK and protobuf generated Python scripts.</p>
+<p class="p-main" >Two modules of DPF Python services are currently available: DPF-Core and DPF-Post. They are both available in open source. DPF-Core is also available for internal development.</p>
+
+<h3 class="p-main" >Open source modules</h3>
+<p class="p-main" id="des_PackageCPythonInstall">Once you've installed Ansys 2021R1 or newer, you can install DPF with:</p>
+<p class="p-main" >This will install the latest version of ``ansys-dpf-core`` and ``ansys-dpf-post`` and all the necessary dependencies.</p>
+
+<h3 class="p-main" >Editable open source install (Development Mode)</h3>
+<p class="p-main" id="des_PackageCPythonOpenSourceDev">If you wish to edit and potentially contribute to the DPF-Core or DPF-Post python
+module, clone the repository and install it using pip with the ``-e`` development flag.</p>
+
+<h2 class="h2-main">Connect to a specific server</h2>
+<p class="p-main" id="des_CPythonConnectServer">
+    User may want to set a server with a specific id and port in order to connect to it from a different machine. Here is the code to proceed: 
+</p>
+<simple-scripts id="script_PackageCPythonInstall" python=" pip install ansys-dpf-core
+ pip install ansys-dpf-post"></apis-scripts><simple-scripts id="script_PackageCPythonOpenSourceDev" python=" git clone https://github.com/pyansys/DPF-Core
+ cd DPF-Core
+ pip install -e .
+
+ git clone https://github.com/pyansys/DPF-Post
+ cd DPF-Post
+ pip install -e ."></apis-scripts><simple-scripts id="script_CPythonConnectServer" python="# start a specified server<br># please note that without attribute, this function <br># will start a server with default id and port <br>from ansys.dpf import core as dpf<br>dpf.start_local_server(ip = my_ip, port = my_port)<br><br># Connect to an existing DPF server<br>from ansys.dpf import core as dpf<br>server = dpf.connect_to_server(ip = my_ip, port = my_port)<br>"></apis-scripts></div></div><div class="main_text" id="howToUseIPython"><h1>How to use the IPython package</h1><div><h2 class="h2-main">ACT Console</h2>
+<h3 class="h3-main">Open the ACT console scripting in Mechanical</h3>
+<p class="p-main" id="des_PackageOpenACT">The ACT console can be opened by clicking on "Automation"/"Scripting" menus in Mechanical.</p>
+<h3 class="h3-main">Import DPF and connect it to the current console</h3>
+<p class="p-main" id="des_PackageImportDPFACT">To import DPF�s capabilities into the ACT console, the user should import mech_dpf to access helpers and import the framework contained in the Ans.DataProcessing module. To access data from the current Mechanical windows, DPF�s helpers should be linked to it through the extAPI.</p>
```

Comment:
To import DPF's capabilities into the ACT console, import mech_dpf to access helpers...

Reply:
updated in TFS